DatadogRUM/Sources/Integrations/CrashReportReceiver.swift

```diff
@@ -203,7 +206,9 @@ internal struct CrashReportReceiver: FeatureMessageReceiver {
             DD.logger.debug("Sending crash as RUM error.")
             core.scope(for: RUMFeature.self).eventWriteContext(bypassConsent: true) { context, writer in
                 let rumError = createRUMError(from: crashReport, and: lastRUMViewEvent, crashDate: crashTimings.realCrashDate, sourceType: context.nativeSourceOverride)
-                writer.write(value: rumError)
+                if let mappedError = self.eventsMapper.map(event: rumError) {
+                    writer.write(value: mappedError)
+                }
```

Comment:
question/ Why do we allow dropping crash in this mapper call? Seems inconsistent with the next crash write in this file. Is it because in this case it goes without corresponding view event? If it's on purpose, I think it may make sense to add code comment on this decision.

Reply:
No it's not on purpose. I'll make sure the test covers this and fix.